homeworks/dankhazipeter/9_fastapi/models.py

```diff
@@ -0,0 +1,30 @@
+
+from sqlalchemy import Column, String, Integer, Float
+from sqlalchemy.orm import Mapped, mapped_column
+from uuid import uuid1, UUID
+from pydantic import BaseModel
+from typing import Optional
+from database import Base
+
+# DATABASE MODEL
+
+
+class Product(Base):
+    __tablename__ = "products"
+    id: Mapped[str] = mapped_column(
```

Comment:
Itt az órán máshogy csináltuk, amire ezt írta az IDE: Type "Column[str]" is not assignable to declared type "Mapped[str]"   "Column[str]" is not assignable to "Mapped[str]"

Ez volt a javaslata, ez alapján írtam át: To fix the type error, you should import and use the [Mapped] type from [sqlalchemy.orm] correctly.